src/js/components/invoice/InvoiceListFilters.jsx

```diff
@@ -111,20 +111,25 @@ const InvoiceListFilters = ({
   const [defaultValues, setDefaultValues] = useState({});
 
   useEffect(() => {
-    const initialEmptyValues = Object.keys(filterFields).reduce((acc, key) => {
-      if (!acc[key]) return { ...acc, [key]: '' };
-      return acc;
-    }, {});
-    setDefaultValues({
-      ...initialEmptyValues,
-      buyerOrganization: {
-        id: currentLocation?.organization?.id,
-        value: currentLocation?.organization?.id,
-        name: currentLocation?.organization?.name,
-        label: currentLocation?.organization?.name,
-      },
-    });
-  }, [currentLocation]);
+    // Avoid unnecessary re-fetches if getAppContext triggers fetching session info
+    // but currentLocation doesn't change
+    // eslint-disable-next-line max-len
```

Comment:
Is it required? Looks like the next line is not that long.

Reply:
Forgot to delete it after refactoring `if` to shorter version